README.md

```diff
@@ -8,3 +8,13 @@ Add your entry in this format to the list below:
 Please don't forget to add a newline after your entry.
 
 # List
+
+HOUSEFINDER - A SaaS that will help you easily look for an apartment/house rental contract via a web crawler that collects the latest info from several websites with rental contracts. Created by Adam Jacobs [ @worldyn ] and Olle Berglöf [ edit comes soon ].
```

Comment:
Could you make HOUSEFINDER into a link maybe? Like so.

```
[HOUSEFINDER](https://github.com/worldyn/house-web-crawler)
```

And the same for the Github usernames.

Will approve and merge when your partner is listed as well 👍